frontend/src/components/samples/SamplesAssociatedProjects.js

```diff
@@ -80,56 +88,58 @@ const SamplesAssociatedProjects = ({
   setSortBy,
 }) => {
 
-  const filters = {}
+  useEffect(() => {
+    // returned function will be called on component unmount
+    return () => {
+      clearFilters()
```

Comment:
Is this behaviour consistent with the other entities list pages?

Reply:
It is not. However, it is necessary because of the filtering behaviour I implemented in order to re-use some components. I can explain it with details if you want.